draft-irtf-cfrg-vdaf.md

```diff
@@ -293,6 +293,16 @@ security considerations for VDAFs.
 
 (\*) Indicates a change that breaks wire compatibility with the previous draft.
 
+04:
+* Change Prio3 spec to include the nonce in joint randomness derivation. This 
+  mitigates a toy attack on robustness where a client submits the same set of 
+  input shares under two distinct nonces and exploits the use of the same joint 
+  randomness in both preparation phases. The mitigated attack has the same query 
+  complexity as a birthday attack.
+
+* Change VDAF syntax to include a nonce as an input in the sharding phase. This
+  accommodates the above change to Prio3. 
```

Comment:
You can remove the change log update; we typically wait to do this until the next draft is cut.

FWIW, I would summarize the Prio3 as follows

> Prio3: Add the nonce to the transcript for joint randomness computation. This improves the provable security bound for robustness.